### PR TITLE
gauge 0.9.3

### PR DESCRIPTION
**Pull request by Apoorva-GA**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/gauge.rb [hunk 1]
```diff
@@ -17,25 +15,16 @@ class Gauge < Formula
...
-        :revision => "f16bcf089e263db525a255c9f9621ca0270ff1d3"
-  end
-
   def install
     ENV["GOPATH"] = buildpath
     ENV["GOROOT"] = Formula["go"].opt_libexec
 
     # Avoid executing `go get`
-    inreplace "build/make.go", /\tgetGaugeScreenshot\(\)\n/, ""
+    inreplace "build/make.go", "", ""
```
Comment (ilovezfs) [inline Formula/gauge.rb]: Why are we replacing the empty string with the empty string?

Comment (fxcoudert) [reply Formula/gauge.rb]: You don't have clearance for that information, obviously

**Comment by ilovezfs** [comment]
@Apoorva-GA please squash this down to one commit.

**Comment by ilovezfs** [comment]
Thanks for your first contribution to Homebrew, @Apoorva-GA! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**